src/m365/spo/commands/user/user-remove.spec.ts

```diff
@@ -119,6 +209,58 @@ describe(commands.USER_REMOVE, () => {
     assert.notStrictEqual(actual, true);
   });
 
+
+  it('fails validation if entraGroupId is not a valid id', async () => {
+    const actual = await command.validate({ options: { webUrl: validWebUrl, entraGroupId: 'invalid' } }, commandInfo);
+    assert.notStrictEqual(actual, true);
+  });
+
+  it('fails validation if id is not a valid id', async () => {
```

Comment:
Maybe something like `...is not valid` is enough or `.. is not a valid number` 😉

```suggestion
  it('fails validation if id is not a valid number', async () => {
```